doc/calibrations/wave_calib.rst

```diff
@@ -259,6 +259,54 @@ observations, long-slit observations where wavelengths
 vary (*e.g.*, grating tilts).  We are likely to implement
 this for echelle observations (*e.g.*, HIRES).
 
+.. _wvcalib-echelle:
+
+Echelle Spectrographs
+=====================
+
+Echelle spectrographs are a special case for wavelength
+solutions, primarily because the orders follow the
+grating equation.
+
+In general, the approach is:
+
+    1. Identify the arc lines in each order
+    2. Fit the arc lines in each order to a polynomial, individually
+    3. Fit a 2D solution to the lines using the order number as a basis
+    4. Reject orders where the RMS of the fit (measured in binned pixels) exceeds ``rms_threshold``
+    5. Attempt to recover the missing orders using the 2D fit and 
+      a higher RMS threshold
+    6. Refit the 2D solution
+
+One should always inspect the outputs, especially the 2D solution
+(global and orders).  One may then need to modify the ``rms_threshold``
+parameter and/or hand-fit a few of the orders to improve the solution.
+
+.. _wvcalib-rms-threshold:
+
+rms_threshold
+-------------
+
+All of the echelle spectrographs have a default ``rms_threshold`` 
+matched to a default ``FWHM`` parameter (also measured in binned pixels).
+The ``rms_threshold`` adopted in the analysis is one 
+scaled by the measured FWHM from the arc lines 
+(again, binned pixels) of the adopted calibration files
+
+That is, each order must satisfy the following:
+
+.. code-block:: ini
+
```

Comment:
My suggestion is that we instead define: 

rms_pixels = rms_threshold*FWHM

Where FWHM is either the default FWHM or the measured FWHM, i.e. it will be the latter when the automatic FWHM determination is turned on.

Reply:
Here is what we are trying to handle:

1. For a given spectrograph, we have a default `rms_threshold` (pixels) corresponding to a default `fwhm` (pixels) which we tune from the DevSuite.  Both of these depend on slit width and binning
2. Now, consider a new exposure with different slit width and/or binning.  The sensible thing is to scale `rms_threshold` by the measured fwhm.  This works relatively well but is not perfect.  Nothing ever is for wavelengths..  This is what is currently implemented in the code and described in the docs.

The `fwhm_fromlines` parameter only refers to whether to use the measured_fwhm when finding arclines.
  It doesn't play a role `rms_threshold`.

Reply:
Please add an issue to updated the docs and spectrograph files to the convention that we discussed, i.e using rms_threshold_frac_fwhm. That need not be done for this PR, but clearly should be done asap.